README.md

```diff
@@ -181,6 +181,20 @@ functionName() => String, throws: Error
 ```
 
 
+### Dependencies
+
+If necessary you can list the non built-in functions required, which is especially useful if you are using polyfills.
```

Comment:
> "If necessary you can list the non built-in functions required"

Would this be easier to understand?

"You can optionally list your function's dependencies. In the future, add on tools may automatically scan your functions and list dependencies for you, which could be useful for documentation and to identify polyfills required for your application to run on browsers and platforms that lack the required features."